client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -155,6 +167,7 @@ export default connect(
 			domainsList: getFlatDomainsList( state ),
 			domainsDetails: getAllDomains( state ),
 			requestingDomains: isRequestingAllDomains( state ),
+			isRequestingAllSiteDomains: getAllRequestingSiteDomains( state ),
```

Comment:
This prop would probably be better named to something like `siteDomainsBeingRequested`, since the current name implies it's a boolean. Other than that looks good 👍

Reply:
To make it a little consistent, I changed it to `requestingSiteDomains`